website/server/models/user/schema.js

```diff
@@ -512,6 +523,7 @@ let schema = new Schema({
   pushDevices: [{
     regId: {type: String},
     type: {type: String},
+    createdAt: {type: Date, default: Date.now},
```

Comment:
for this it would be better to use a subschema with timestamps like we do for tags